composer.json

```diff
@@ -34,5 +31,5 @@
             "dev-master": "0.1-dev"
         }
     },
-    "bin": ["symfony-upgrade-fixer"]
+    "bin": ["bin/symfony-upgrade-fixer"]
```

Comment:
@lyrixx Why is it moved to bin? This project extends [PHP-CS-Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer), I tend to keep same structure to make contribution easier.